### PR TITLE
chore: upgrade react-native-reanimated to 2.9.1

### PR DESCRIPTION
**Pull request by J-Son89**
This pr upgrades the version of `react-native-reanimated` to `2.9.1` as this version (or higher) is required for the Parallax component to be added.

Creating this pr to keep the scope of the parallax pr smaller and focused when it is eventually opened.

Also it includes the new dependency - `react-native-transparent-video`, created by @briansztamfater into the repo.

I used the command `nix-update-gradle` for this.

### CODE REVIEW

**Comment by briansztamfater** [comment]
Maybe better to exclude `react-native-transparent-video` from this PR as we don't need it now, and add it on #16189 when we can assure is stable enough?

File: ios/Podfile.lock [hunk 1]
```diff
@@ -380,15 +382,14 @@ PODS:
...
-  - RNReanimated (2.3.3):
+  - RNReanimated (2.9.1):
     - DoubleConversion
     - FBLazyVector
     - FBReactNativeSpec
     - glog
     - RCT-Folly
     - RCTRequired
     - RCTTypeSafety
-    - React
```
Comment (OmarBasem) [inline ios/Podfile.lock:391]: Why is `React` removed from `Podfile.lock` ?

Comment (siddarthkay) [reply ios/Podfile.lock:391]: It's autogenerated and React is removed as a dep from re-animated lib and not the whole project soo this should be fine 👍🏻

**Review by siddarthkay — APPROVED** [review]

**Review by jakubgs — APPROVED** [review]

File: ios/Podfile.lock (hunk 1 above)
Comment (J-Son89) [reply ios/Podfile.lock:391]: thanks @siddarthkay :)

**Comment by J-Son89** [comment]
> Maybe better to exclude `react-native-transparent-video` from this PR as we don't need it now, and add it on #16189 when we can assure is stable enough?

@briansztamfater , I think it's okay because we are not using the library yet, it will reduce some noise in the follow up pr's this way. Or is this a blocker for you for me to merge this pr?

**Comment by J-Son89** [comment]
removed change with react-native-transparent-video from this pr 👍

**Comment by qoqobolo** [comment]
@J-Son89 adding here an issue about the weird bottom sheet behavior as discussed 

### ISSUE 1: Slow bottom sheet response when pressing the `plus` button on iOS 

**Steps:**
1. Just press the `plus` button on the home screen
or
2. Log out > tap the Profiles icon > Tap the `plus` button

https://github.com/status-im/status-mobile/assets/67952253/59f77118-3cf7-4da9-99cf-037151104a2c

Screenshot 2023-06-08 at 18 12 39

**Comment by J-Son89** [comment]
Thanks @qoqobolo, I am going to try a later version and see if performance improves

**Comment by J-Son89** [comment]
I will close this pr for now as @siddarthkay recently upgraded react-native-reanimated beyond 2.9.1 which includes the functionality needed for parallax 👍

**Comment by J-Son89** [comment]
I will close this pr for now as @siddarthkay recently upgraded react-native-reanimated beyond 2.9.1 which includes the functionality needed for parallax 👍

**Pull request closed without merging**